backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -157,7 +157,7 @@ const createEmailOptIn = async (
   })
   if (emailOptIn) {
     if (isOptInCodeValid(emailOptIn)) {
-      throw new Error(`email already sent less than $(CONFIG.EMAIL_CODE_VALID_TIME} minutes ago`)
+      throw new Error(`email already sent less than ${printEmailCodeValidTime()} ago`)
```

Comment:
this must be a different parameter, which still is 10mins, since if the user loses his code he can not request another for 24h with this code